app/config/platforms.php

```diff
@@ -81,7 +81,7 @@
             [
                 'key' => 'apple',
                 'name' => 'Apple',
-                'version' => '1.2.0',
+                'version' => '1.2.1',
```

Comment:
I forgot to commit previous release versions:

```suggestion
                'version' => '1.2.3',
```